resources_portal/models/material.py

```diff
@@ -10,13 +10,23 @@ class Meta:
         get_latest_by = "created_at"
 
     objects = models.Manager()
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
 
+    material_type = models.CharField(max_length=32)
```

Comment:
Can we use [Django's choices](https://docs.djangoproject.com/en/3.0/ref/models/fields/#choices) here to enumerate the valid values and make sure that we only ever store those. I don't like the way the django choices feature has the "human readable" version and then the version that actually gets stored in the DB, so can you just make them the same, i.e. `]("CELL_LINE", "CELL_LINE"), ...]`

Reply:
Added the following choices:

```
    MATERIAL_TYPE_CHOICES = (
        ("CELL_LINE", "CELL_LINE"),
        ("PLASMID", "PLASMID"),
        ("PROTOCOL", "PROTOCOL"),
        ("DATASET", "DATASET"),
        ("MOUSE_MODEL", "MOUSE_MODEL"),
        ("ZEBRAFISH_MODEL", "ZEBRAFISH_MODEL"),
    )
```